api/health/client.go

```diff
@@ -17,11 +17,11 @@ var _ Client = (*client)(nil)
 // AwaitHealthy, and AwaitAlive.
 type Client interface {
 	// Readiness returns if the node has finished initialization
-	Readiness(context.Context, ...rpc.Option) (*APIReply, error)
+	Readiness(ctx context.Context, tags []string, options ...rpc.Option) (*APIReply, error)
 	// Health returns a summation of the health of the node
-	Health(context.Context, ...rpc.Option) (*APIReply, error)
+	Health(ctx context.Context, tags []string, options ...rpc.Option) (*APIReply, error)
 	// Liveness returns if the node is in need of a restart
-	Liveness(context.Context, ...rpc.Option) (*APIReply, error)
+	Liveness(ctx context.Context, tags []string, options ...rpc.Option) (*APIReply, error)
```

Comment:
We use `...string` in all the other parts of the code but we use `[]string` here... Should we stick with `...string`?

Reply:
I don't think we can since we already have `...rpc.Option`

Reply:
ah duh